RHDEVDOCS-3930 - Added 1.4.5 release notes (#44078)
openshift/openshift-docs: modules/gitops-release-notes-1-4-5.adoc

// Module included in the following assembly:
//
// * gitops/gitops-release-notes.adoc

[id="gitops-release-notes-1-4-5_{context}"]
= Release notes for {gitops-title} 1.4.5

[role="_abstract"]
{gitops-title} 1.4.5 is now available on {product-title} 4.7, 4.8, and 4.9.

> 

Reviewer: If the release is supported on 4.10, then 4.10 needs to mentioned here as well.

> 

Reviewer 2: @wtam2018 are 1.3 and 1.4 supported on and compatible with 4.10? If yes, then we will need to update the first 2 rows of the compatibility matrix here: https://deploy-preview-44078--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/gitops-release-notes#compatibility-and-support-matrix

> 

Reviewer 2: Post discussions with QE, since docs cover only the major releases in the matrix table, to avoid confusion to the end users, we will add the versions supported in the major releases as per the component matrix maintained by the team.

[id="fixed-issues-1-4-5_{context}"]
== Fixed issues

The following issue has been resolved in the current release:

> [WARNING]

Reviewer: Consider moving the "Warning" admonition up above the intro sentence (so between `== Fixed issues` and `The following issue has been resolved in the current release:`) sentence because the admonition applies generally and not just to the specific issue you mention.

====

> Red Hat strongly recommends to directly upgrade to {gitops-title} v1.4.5 from {gitops-title} v1.4.3. Do not use {gitops-title} v1.4.4 in the production environment. Some major issues that affected {gitops-title} v1.4.4 were fixed in {gitops-title} 1.4.5. 

Reviewer: ```suggestion
You should directly upgrade to {gitops-title} v1.4.5 from {gitops-title} v1.4.3. Do not use {gitops-title} v1.4.4 in a production environment.  Major issues that affected {gitops-title} v1.4.4 are fixed in {gitops-title} 1.4.5. 
```
- Per the RH SSG: Avoid "recommends". Instead of "Red Hat recommends", direct users to take the recommended action. This allows Red Hat to be more prescriptive in documentation and prevent any user uncertainty, and is easier for upstream/downstream coordinated efforts. For example, instead of "Red Hat recommends using X package because", write "Use this package because" or "Use this package when".
- Change the definite article "the" to "a" because you're talking generally about any production environment and not one specific production environment.
- Change the second sentence to present tense.

====

> * Before this update, multiple pods were stuck in the `ErrImagePullBackOff` state. The following error message was shown:

Reviewer: The way it's stated here, it sounds like multiple pods were always stuck in this error state. Is this true? Or did it only happen sometimes under specific circumstances? If the latter, then re-word this sentence for accuracy.

Author: @iam-veeramalla please suggest

Reviewer 3: it happens every time with v1.4.4

Reviewer 3: instead of multiple pods, you can say `argocd pods`
